Code/Editor/EditorViewportCamera.cpp

```diff
@@ -136,6 +136,43 @@ namespace SandboxEditor
         }
     }
 
+    AZStd::optional<AZ::Transform> CalculateGoToEntityTransform(
+        const AZ::Transform& cameraTransform, const float fovRadians, const AZ::Vector3& center, const float radius)
+    {
+        // do not attempt to interpolate to where we currently are
+        if (cameraTransform.GetTranslation().IsClose(center))
+        {
+            return AZStd::nullopt;
+        }
+
+        const AZ::Vector3 forward = [&center, &cameraTransform]
+        {
+            const AZ::Vector3 forward = (center - cameraTransform.GetTranslation()).GetNormalized();
+            // if the camera is looking directly up or down, pitch the camera down or up respectively to avoid
+            // a singularity when creating the lookat transformation below
+            if (const float forwardDot = forward.Dot(AZ::Vector3::CreateAxisZ()); AZ::IsCloseMag(AZ::Abs(forwardDot), 1.0f, 0.001f))
+            {
+                return AZ::Transform::CreateFromQuaternion(
+                           AZ::Quaternion::CreateFromAxisAngle(cameraTransform.GetBasisX(), AZ::DegToRad(5.0f) * -AZ::GetSign(forwardDot)))
+                    .TransformVector(AZ::Vector3::CreateAxisZ() * AZ::GetSign(forwardDot));
+            }
+            return forward;
+        }();
+
+        // minimum center size is 40cm, maxium center size is 25m
+        const float MinSelectionRadius = 0.4f;
+        const float MaxSelectionRadius = 25.0f;

```

Comment:
Since these are local variables they should start with lowercase:
`MinSelectionRadius` -> `minSelectionRadius`
`MaxSelectionRadius` -> `maxSelectionRadius`

Reply:
This is actually allowed and I prefer it in this case (as these are literal values being set) - See https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#constant-names).